packages/backend-core/src/events/publishers/backup.ts

```diff
@@ -10,3 +16,16 @@ export async function appBackupRestored(backup: AppBackup) {
 
   await publishEvent(Event.APP_BACKUP_RESTORED, properties)
 }
+
+export async function appBackupTriggered(
+  appId: string,
+  trigger: AppBackupTrigger,
+  name?: string
+) {
+  const properties: AppBackupTriggeredEvent = {
+    appId: appId,
+    backupName: name,
```

Comment:
The name is probably something we don't need in here, generally anything user specific that isn't defined in our own domain like an id or a constant we should leave out of these

Reply:
Is it really a negative to have this information if the user specified a specific name for the backup? Do we need to be optimising for event size?

Reply:
Not that we need to optimise for event size, more that we don't pull any unnecessary personal data into analytics. 

The rationale I normally use when adding a field is
- Will I use this to do a filter?
- Will I use this to do a group by?
- etc

User entered input doesn't add any valuable insights in most cases. If there was a specific condition we wanted, like "has the user entered a custom name?", it'd be better to work that out application side and set a boolean `customName`, that would be easier to work with.